File: Kode C++/prime.cpp
#include <iostream>
using namespace std;

int main() {
    cout << "Program mengcek bilangan Prima" << endl;

    int bil;
    bool bilPrima = true;

    cout << "masukkan bilangan: ";
    if (!(cin >> bil)) {
        cerr << "Input tidak valid." << endl;
        return 1;
    }

    if (bil < 2) {
        bilPrima = false;
    } else {
        int k = 2;

        while (k <= bil - 1 && bilPrima) {
            if (bil % k == 0) {
                bilPrima = false;
            }

            if (bilPrima) {
                k++;
            }
        }
    }

    if (bilPrima) {
        cout << "Bilangan prima" << endl;
    } else {
        cout << "Bukan bilangan prima" << endl;
    }

    return 0;
}

File: prime-numbers/prime.py
print("Program Cek Bilangan Prima")

bil = int(input("Masukkan bilangan: "))

if bil < 2:
    print("Bukan bilangan prima")
else:
    bilPrima = True
    k = 2

    while k <=bil - 1 and bilPrima:
        if bil % k == 0:
            bilPrima = False
        else:
            k += 1

    if bilPrima:
        print("Bilangan prima")
    else:
        print("Bukan bilangan prima")
